vnext/Microsoft.ReactNative/Modules/Animated/AnimationDriver.cpp

```diff
@@ -54,9 +54,9 @@ void AnimationDriver::StartAnimation() {
   }
   scopedBatch.End();
 
-  m_scopedBatchCompletedToken = scopedBatch.Completed([&](auto sender, auto) {
-    DoCallback(true);
-    if (auto manager = m_manager.lock()) {
+  m_scopedBatchCompletedToken = scopedBatch.Completed([&, weakManager = m_manager](auto sender, auto) {
```

Comment:
yikes this shouldn't capture implicitly by reference if it is async. Would be great if you can remove that part if you are here.

Reply:
Made AnimatedDriver shared_from_this and passed a weak instance to the lambda.